lib/include/ert/util/util.h

```diff
@@ -99,10 +99,10 @@ typedef enum {left_pad   = 0,
               center_pad = 2} string_alignement_type;
 
   //#define UTIL_CXX_MALLOC(var , num_elm) (typeof (var)) util_malloc( (num_elm) * sizeof var)
-
   void         util_bitmask_on(int *  , int );
   char       * util_get_timezone(void);
   time_t       util_make_datetime_utc(int , int , int , int , int , int );
+  bool         util_make_datetime_utc__(int sec, int min, int hour , int mday , int month , int year, time_t * t);
```

Comment:
Well - the function name might not be very good; but there is some consistency to it:

- `util_make_xxxx` : `(year, day, month , [hour,min,sec]) -> time_t`
- `util_set_xxx` : `time_t -> (year , month , day , [hour , min, sec])`

Where the `util_set_xxx` use pointer references to "return" multiple values.